hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/SnapshotDeletingService.java

```diff
@@ -257,20 +276,19 @@ public BackgroundTaskResult call() throws Exception {
               }
               deletionCount++;
             }
-            // Submit Move request to OM.
-            submitSnapshotMoveDeletedKeys(snapInfo, toReclaimList,
-                toNextDBList, renamedKeysList);
 
             // Delete keys From deletedTable
             processKeyDeletes(keysToPurge, omSnapshot.getKeyManager(),
                 snapInfo.getTableKey());
-            snapshotLimit--;
             successRunCount.incrementAndGet();
           } catch (IOException ex) {
             LOG.error("Error while running Snapshot Deleting Service", ex);
           }
```

Comment:
Print more helpful messages in this case. e.g. which snapshotId, how many keys and dirs processed at this point